library/modules/es7.math.fscale.js

```diff
@@ -0,0 +1,40 @@
+// https://www.npmjs.com/package/ecma-proposal-math-extensions
+var $export = require('./_export')
+  // , scale = require('./es7.math.scale')
+  , f32A = new Float32Array(1);
```

Comment:
Hmm... Stop... Looks like [`Math.fround` implementation](https://github.com/zloirock/core-js/blob/master/modules/es6.math.fround.js) can be used in this case.

> If so, that means I need to extract `./_scale.js`, too, right?

Yep. But `./_math-scale.js`.